2011.final-value-of-variable-after-performing-operations.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * https://leetcode.com/problems/final-value-of-variable-after-performing-operations/description/
+ */
+
+/**
+ * @param {string[]} operations
+ * @return {number}
+ */
+var finalValueAfterOperations = function (operations) {
+  let result = 0;
+  operations.map((x) => (x.includes("+") ? (result += 1) : (result -= 1)));
```

Comment:
```suggestion
  operations.map((x) => x.includes("+") ? result += 1 : result -= 1);
```
you can remove some parentheses in map()

Reply:
Sure. But those parentheses were formatted because of prettier.
I'll modify those anyway. Thank you.